src/nginxconfig/templates/global_sections/logging.vue

```diff
@@ -33,11 +33,36 @@ THE SOFTWARE.
             <div class="field-body">
                 <div class="field">
                     <div :class="`control${accessLogChanged ? ' is-changed' : ''}`">
+                        <div class="checkbox">
+                            <PrettyCheck v-model="accessLog" class="p-default p-curve p-fill p-icon">
+                                {{ $t('common.enable') }}
+                            </PrettyCheck>
+                        </div>
+                    </div>
+                    <div v-if="$props.data.accessLog.computed" class="control field is-horizontal is-expanded">
+                        <input
+                            v-model="accessLogPath"
+                            class="input"
+                            type="text"
+                            :placeholder="$props.data.accessLogPath.default"
+                        />
+                    </div>
```

Comment:
Ditto on all three thoughts here (is-changed indicator, watcher for the default computed, parent field alignment)